src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  const discountLongRental = 50;
+
+  const discountShortRental = 20;
+
+  if (days >= 7) {
+    const newTotalCost = totalCost - discountLongRental;
+
+    return newTotalCost;
+  }
+
+  if (days >= 3) {
```

Comment:
Use a constant to represent the minimum number of days for a short-term rental instead of the hardcoded value `3`. This will make the code more understandable and easier to change if the policy changes.